components/raftstore/src/store/snap.rs

```diff
@@ -577,7 +577,7 @@ impl Snapshot {
     }
 
     fn get_display_path(dir_path: impl AsRef<Path>, prefix: &str) -> String {
-        let cf_names = "(".to_owned() + &SNAPSHOT_CFS.join("|") + ")";
+        let cf_names = "(".to_owned() + SNAPSHOT_CFS.join("|").as_str() + ")";
```

Comment:
Why the change?

Reply:
I don't even know; some compile errors just pop up even though master works fine.